tools/wptrunner/wptrunner/tests/test_products.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+
+from os.path import join, dirname
+
+import pytest
+
+sys.path.insert(0, join(dirname(__file__), "..", ".."))
+
+from wptrunner import products
+
+current_tox_env = None
+current_tox_env_split = None
+if "CURRENT_TOX_ENV" in os.environ:
+    current_tox_env = os.environ["CURRENT_TOX_ENV"]
+    current_tox_env_split = current_tox_env.split("-")
+
+product_env_map = {
+    "chrome_android": "chrome",
+    "servodriver": "servo",
+}
+
+@pytest.mark.parametrize("product", [
```

Comment:
Right, so if we keep doing this like this, we should create a test helper that returns the list of active products or the full list, and parameterise over that.

Reply:
I think in this case it actually makes sense, because we actually want to test something more: we want to check that it either that the product loads successfully or throws ImportError, and any other result should be a failure.

Reply:
I've laid the groundwork to make this easy in future.